cpp/src/stream_compaction/distinct_helpers.hpp

```diff
@@ -87,8 +87,8 @@ using hash_set_type =
  * @param mr Device memory resource used to allocate the returned vector
  * @return A device_uvector containing the output indices
  */
-template <typename RowHasher>
-rmm::device_uvector<size_type> reduce_by_row(hash_set_type<RowHasher>& set,
+template <typename RowEqual>
```

Comment:
Can we please add the missing `@tparam RowEqual ` to the docstring of this function

Reply:
Good catch! Doc added